### ISSUE TITLE
better docs on how to contribute and test

### ISSUE THREAD

**Issue by woloski** [tjanczuk/git-azure#47]
We should add a wiki page explaining how to fork the project and run your own instance of git-azure.

In essence:
- Fork git-azure and do whatever change you want and push to your fork
- Create a test repo to try things
- DON'T run git azure init yet (you want to first add your .git-azure submodule)
- Add your own git-azure submodule `git submodule add git@youruser:git-azure .git-azure`
- Run git azure init ...
- Pull request :)

The way this works is because `git azure init` won't bring the original git-azure from https://github.com/tjanczuk/git-azure since there is already a .git-azure folder in your repo, so it will skip that and you will be running with your own

**Comment by tjanczuk** [comment]
Good idea about wiki. I wonder if we should just add an option to `git azure init` to specify the URL of the .git-azure repository to create the submodule of off. 

---

From: Matias Woloski [reply@reply.github.com]
Sent: Saturday, June 23, 2012 9:12 AM
To: Tomasz Janczuk
Subject: [git-azure] better docs on how to contribute and test (#47)

We should add a wiki page explaining how to fork the project and run your own instance of git-azure.

In essence:
- Fork git-azure and do whatever change you want and push to your fork
- Create a test repo to try things
- DON'T run git azure init yet (you want to first add your .git-azure submodule)
- Add your own git-azure submodule `git submodule add git@youruser:git-azure .git-azure`
- Run git azure init ...
- Pull request :)

---

Reply to this email directly or view it on GitHub:
https://github.com/tjanczuk/git-azure/issues/47

**Comment by woloski** [comment]
that would be even better
